source/index.slim

```diff
@@ -20,7 +20,13 @@ nav_logo_class: 'is-animated'
 
 - content_for :javascripts
   javascript:
-    $('#HomepageHero').load(function() {
-      var svg = $('#HomepageHero').contents().find('svg')[0];
-      window.homepage.animateLines(svg);
-    })
+    var heroWrapper = $('#HomepageHero');
+    var objectElement = $('#HomepageHeroSVG');
+
+    objectElement
+      .on('load', function() {
```

Comment:
Is it possible to keep the `.` in the previous line? We do that in ruby because we feel it's easier to understand that it's part of the previous line that way.